### ISSUE TITLE
Ltac redefs (::=) are not indented properly

### ISSUE THREAD

**Issue by jonleivent** [ProofGeneral/PG#91]
Redefinition of Ltac tactics via "::=" should get indented the same way as original defs (":=").

**Comment by cpitclaudel** [comment]
Yup, I noticed this one too :) @Matafou, here's an example:

```coq
Ltac a :=
  idtac.

Ltac a ::=
    fail. (* Notice the extra indentation *)
```

**Comment by Matafou** [comment]
Should be fixed by 	f8a4f52.
